situp-api/src/test/java/com/amazon/situp/model/record/RecordTests.java

```diff
@@ -9,30 +9,47 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
 
 public class RecordTests {
     private static final String TEST_DATA = "TEST";
     private static final String TEST_RECORD_TYPE = "OTEL";
     private static final String TEST_RECORD_METADATA_V_TYPE = "VERSION";
+    private static final String UNKNOWN = "unknown";
+    private static final String ATTRIBUTE_NOT_PRESENT = "NOT_PRESENT";
     private static final String TEST_RECORD_VERSION = "1.0";
 
     @Test
     public void testRecordOperations() {
         final Record<String> stringRecord = new Record<>(TEST_DATA);
-        final Record<String> stringRecordWithDefaultMetadata = new Record<>(TEST_DATA, RecordMetadata.defaultMetadata());
-        assertThat("Incorrect Record data is returned", stringRecord.getData(), is(equalTo(TEST_DATA)));
-        assertThat("Incorrect Record data is returned", stringRecordWithDefaultMetadata.getData(),
-                is(equalTo(TEST_DATA)));
-        final RecordMetadata defaultRecordMetadataWithoutPassing = stringRecord.getMetadata();
-        final RecordMetadata defaultRecordMetadataWithPassing = stringRecordWithDefaultMetadata.getMetadata();
-        assertThat("Default metadata should match",
-                defaultRecordMetadataWithoutPassing.getMetadataObject(),
-                is(equalTo(defaultRecordMetadataWithPassing.getMetadataObject())));
-        assertThat("Incorrect default metadata for record",
-                defaultRecordMetadataWithoutPassing.getAsString(RECORD_TYPE), is(equalTo("unknown")));
-        final Map<String, Object> metadataObject = defaultRecordMetadataWithoutPassing.getMetadataObject();
-        assertThat("Incorrect record metadata object values", metadataObject.get(RECORD_TYPE),
-                is(equalTo("unknown")));
+        assertThat(stringRecord.getData(), is(equalTo(TEST_DATA)));
+        final RecordMetadata recordMetadata = stringRecord.getMetadata();
+        assertThat(recordMetadata.getAsString(RECORD_TYPE), is(equalTo(UNKNOWN)));
+        final Map<String, Object> metadataObject = recordMetadata.getMetadataObject();
+        assertThat(metadataObject.get(RECORD_TYPE), is(equalTo(UNKNOWN)));
+        assertThat(recordMetadata.getAsString(ATTRIBUTE_NOT_PRESENT), nullValue());
+    }
+
+    @Test
+    public void testRecordWithMetadata() {
```

Comment:
This is a bit of a nit, but typically on tests I'll spell out what exactly i'm testing through a snake case method, e.g. `test_record_with_default_metadata_returns_expected_details`.

Reply:
will update them in next PR ?